src/consensus/pbft/pbfttypes.h

```diff
@@ -17,6 +17,7 @@ namespace pbft
   enum PbftMsgType : Node2NodeMsg
   {
     pbft_message = 1000,
```

Comment:
I believe this can start from 0 here since PBFT and Raft are never run at the same time?

Reply:
Both types get compiled however (included in nodeconnections) so they can still get mixed up.